build.sbt

```diff
@@ -47,5 +47,5 @@ lazy val root = (project in file(".")).
   enablePlugins(BuildInfoPlugin).
   settings(
     buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion),
-    buildInfoPackage := "com.fortysevendeg.android.sqlite"
+    buildInfoPackage := "com.fortysevendeg.mvessel"
```

Comment:
You should change the sbt name to `mvessel` as well.

https://github.com/47deg/mvessel/blob/fede-refactor-name/build.sbt#L11